com.netease.CloudMusic.json

```diff
@@ -69,55 +85,65 @@
         "rm -rf /app/share/vlc"
       ]
     },
-    
+
     {
-    	"name": "CloudMusic",
-    	"only-arches": ["x86_64"],
-    	"buildsystem": "simple",
-    	"build-commands": [
-    		"ln -s /app/extra/netease-cloud-music/usr/bin/netease-cloud-music /app/bin/netease-cloud-music",
-    		
-    		"install -D -m0644 -t /app/share/icons/hicolor/scalable/apps/ com.netease.CloudMusic.svg",
-    		"install -D -m0644 -t /app/share/applications/ com.netease.CloudMusic.desktop",
-    		"install -D -m0644 -t /app/share/metainfo/ com.netease.CloudMusic.appdata.xml",
-    		
-    		"install -D -m0755 -t /app/bin/ apply_extra",
+      "name": "CloudMusic",
+      "only-arches": ["x86_64"],
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D -m0755 -t /app/bin/ netease-cloud-music",
 
-    		"cp /usr/bin/ar /app/bin/",
-    		"cp /usr/lib/x86_64-linux-gnu/libbfd* /app/lib/"
-    		
-    	],
-    	"sources": [
-    		{
-    			"type": "extra-data",
-    			"url": "http://d1.music.126.net/dmusic/netease-cloud-music_1.1.0_amd64_ubuntu.deb",
-    			"sha256": "a256ab183513abcaf26279700aaefb8f6a4da89d9d0a498d35a95d5a7d3a345e",
-    			"filename": "netease-cloud-music.deb",
-    			"size": 68515076
-    		},
-    		{
-    		  "type": "script",
-    		  "commands": [
-    		    "mkdir netease-cloud-music",
-    		    "ar x netease-cloud-music.deb",
-    		    "tar xf data.tar.xz --directory=netease-cloud-music",
-    		    "rm -f netease-cloud-music.deb data.tar.xz control.tar.gz debian-binary"
-    		  ],
-    		  "dest-filename": "apply_extra"
-    		},
-    		{
-    			"type": "file",
-    			"path": "com.netease.CloudMusic.desktop"
-    		},
-    		{
-    			"type": "file",
-    			"path": "com.netease.CloudMusic.svg"
-    		},
-    		{
-    			"type": "file",
-    			"path": "com.netease.CloudMusic.appdata.xml"	
-    		}
-    	]
+        "install -D -m0644 -t /app/share/icons/hicolor/scalable/apps/ com.netease.CloudMusic.svg",
+        "install -D -m0644 -t /app/share/applications/ com.netease.CloudMusic.desktop",
+        "install -D -m0644 -t /app/share/metainfo/ com.netease.CloudMusic.appdata.xml",
```

Comment:
Please try using `${FLATPAK_DEST}` and  like this `${FLATPAK_DEST}/share/applications/` and `${FLATPAK_ID}` for these three.